app/assets/stylesheets/partials/wice_grid.scss

```diff
@@ -126,9 +113,9 @@
   ul {margin-left: 0 }
 
   a.wice-grid-delete-query .delete-icon{
-    background: transparent url(/assets/icons/grid/delete.png) no-repeat;
-    float: left;
     @include icon-dimensions;
+    background: transparent asset-url("/assets/icons/grid/delete.png") no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.